### PR TITLE
Added speedtest

### PR DESCRIPTION
**Pull request by Iikkas**

### CODE REVIEW

File: speedtest.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const FastSpeedtest = require("fast-speedtest-api");
+
+////////////////////
+// Fast speedtest //
+////////////////////
+
+const nextDate = new Date();
```
Comment (Steellow) [inline speedtest.js:7]: Unused variable, should remove

**Review by Steellow — CHANGES_REQUESTED** [review]
1 lil change

File: speedtest.js (hunk 1 above)
Comment (Iikkas) [reply speedtest.js]: done

**Review by Steellow — APPROVED** [review]
lg2m

**Pull request merged**